include/arch/riscv/arch/32/mode/machine.h

```diff
@@ -16,16 +16,24 @@
  * Currently all RISC-V 32-bit platforms supported have the mtime register
  * mapped at the same offset of the base address of the CLINT.
  */
-#define CLINT_MTIME_OFFSET 0xbff8
+#define CLINT_MTIME_OFFSET_LO 0xbffc
+#define CLINT_MTIME_OFFSET_HI 0xbff8
```

Comment:
Are these swapped around? IE LO should be `0xbff8` and HI should be `0xbffc`

Reply:
Turns out I have had a misconception about endianness, my bad.

Reply:
So it works now?

Reply:
Yes

Reply:
The CI for Spike 32-bit will confirm this.